R/plotAbundanceDensity.R

```diff
@@ -51,31 +40,30 @@
 #'   
 #' @param ... additional parameters for plotting. 
 #' \itemize{
-#'   \item{xlab}{ a single character value for selecting the x-axis label. 
-#'   (default: \code{xlab = assay.type}) }
+#'   \item \code{xlab} \code{Character scalar}. Selects the x-axis label. 
+#'   (Default: \code{assay.type})
 #'   
-#'   \item{ylab}{ a single character value for selecting the y-axis label. 
+#'   \item \code{ylab} \code{Character scalar}. Selects the y-axis label. 
 #'   \code{ylab} is disabled when \code{layout = "density"}. 
-#'   (default: \code{ylab = "Taxa")} }
+#'   (Default: \code{"Taxa"})
 #'   
-#'   \item{point_alpha}{ a numeric value from range 0 to 1 selecting the transparency of
-#'   colour in \code{jitter} and \code{point} plot. (default: \code{point_alpha = 0.6}) }
+#'   \item \code{point_alpha} \code{Numeric scalar}. From range 0 to 1. Selects the transparency of
+#'   colour in \code{jitter} and \code{point} plot. (Default: \code{0.6})
 #'   
-#'   \item{point_shape}{ a positive integer value selecting the shape of point in
-#'   \code{jitter} and \code{point} plot. (default: \code{point_shape = 21}) }
+#'   \item \code{point_shape} \code{Positive integer scalar}. Value selecting the shape of point in
+#'   \code{jitter} and \code{point} plot. (Default: \code{21})
 #'   
-#'   \item{point_size}{ a positive numeric value selecting the size of point in
-#'   \code{jitter} and \code{point} plot. (default: \code{point_size = 2}) }
+#'   \item \code{point_size} \code{Positive integer scalar}. Selects the size of point in
+#'   \code{jitter} and \code{point} plot. (Default: \code{2})
 #'   
-#'   \item{add_legend}{ a boolean value selecting if legend is added. 
-#'   (default: \code{add_legend = TRUE}) }
+#'   \item \code{add_legend} \code{Logical scalar}. Determines if legend is added. 
+#'   (Default: \code{TRUE})
 #'   
```

Comment:
These should be point.shape (parameter.name)?

Reply:
Thanks.